[2022-02-23 21:38:55,394] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: tutorial.pre.pre_get_data scheduled__2022-02-21T17:28:01.321646+00:00 [queued]>
[2022-02-23 21:38:55,422] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: tutorial.pre.pre_get_data scheduled__2022-02-21T17:28:01.321646+00:00 [queued]>
[2022-02-23 21:38:55,423] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-23 21:38:55,425] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-23 21:38:55,426] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-23 21:38:55,452] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): pre.pre_get_data> on 2022-02-21 17:28:01.321646+00:00
[2022-02-23 21:38:55,462] {standard_task_runner.py:52} INFO - Started process 137 to run task
[2022-02-23 21:38:55,467] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'tutorial', 'pre.pre_get_data', 'scheduled__2022-02-21T17:28:01.321646+00:00', '--job-id', '115', '--raw', '--subdir', 'DAGS_FOLDER/test_dag.py', '--cfg-path', '/tmp/tmph2rusi7t', '--error-file', '/tmp/tmpz2gvzbu9']
[2022-02-23 21:38:55,469] {standard_task_runner.py:77} INFO - Job 115: Subtask pre.pre_get_data
[2022-02-23 21:38:55,581] {logging_mixin.py:109} INFO - Running <TaskInstance: tutorial.pre.pre_get_data scheduled__2022-02-21T17:28:01.321646+00:00 [running]> on host ff48034dfdbf
[2022-02-23 21:38:55,688] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=tutorial
AIRFLOW_CTX_TASK_ID=pre.pre_get_data
AIRFLOW_CTX_EXECUTION_DATE=2022-02-21T17:28:01.321646+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-02-21T17:28:01.321646+00:00
[2022-02-23 21:39:55,681] {python.py:175} INFO - Done. Returned value was: None
[2022-02-23 21:39:55,697] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=tutorial, task_id=pre.pre_get_data, execution_date=20220221T172801, start_date=20220223T213855, end_date=20220223T213955
[2022-02-23 21:39:55,746] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-23 21:39:55,795] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
